Tests/UnitTests/Purchasing/ConfigurationTests.swift

```diff
@@ -35,19 +35,18 @@ class ConfigurationTests: TestCase {
         let configuration = Configuration.Builder(withAPIKey: "test").build()
 
         expect(configuration.observerMode) == false
-        expect(configuration.storeKit2Setting) == .enabledOnlyForOptimizations
+        expect(configuration.storeKitVersion) == .storeKit1
```

Comment:
Let's leave 2 extra tests (marked as deprecated) that test the `usesStoreKit2IfAvailable` configuration. Since it's still available, we need to make sure that it continues to work.

Edit: never mind I see you did that :D